### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by tchan408**
Fixed 1-click Heroku deployment.
Removed unnecessary HOSTNAME parameter.

### CODE REVIEW

**Comment by lornajane** [comment]
What's a good way to understand and test these changes? Just read the readme/docs and try to use the tool? Other advice gratefully received :+1:

**Comment by lornajane** [comment]
OK I got some insider information and have seen it working locally today. I'll try to find time later in the week to test the Heroku change too.

File: app.json [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
-  "description": "The Lex Connector allows a Vonave Voice API call to be connected to any AWS Lex bot. This is the bridge between voice and the bot functionality that allows users to 'talk' directly with a bot.",
+  "name": "Lex Connector - Connect Vonage calls to AWS Lex bots",
+  "description": "The Lex Connector allows a Vonage Voice API call to be connected to any AWS Lex bot. This is the bridge between voice and the bot functionality that allows users to 'talk' directly with a bot.",
   "keywords": ["lex", "bot", "python", "amazon web services", "vonage"],
   "website": "https://developer.nexmo.com/extend/amazon-lex-connector",
-  "repository": "https://github.com/Nexmo/lex-connector"
+  "repository": "https://github.com/Nexmo/lex-connector",	
+  "env":{
+    "NAME": {
+      "description": "The Heroku app name, enter it exactly the same as you have above"
```
Comment (lornajane) [inline app.json:9]: I think the description needs an update here

File: heroku [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+	You are being <a href="https://heroku.com/deploy?template=https://github.com/tchan408/lex-connector/blob/patch-2">redirected</a>
```
Comment (lornajane) [inline heroku:3]: Does this need updating for us to merge this to master?

**Review by lornajane — CHANGES_REQUESTED** [review]
Looks good and will greet me if I tell her my name both locally and on Heroku! I have a couple of minor comments that I think need tidying up but let's get those straightened out and I'll be happy to merge once that's done.

**Comment by lornajane** [comment]
Can we add some examples of what she can do as well please? I got some insider help to say "Hello, my name is Bob" to her, but what else would be good for someone using this for the first time? Would be good to have something in the README

File: app.json (hunk 1 above)
Comment (tchan408) [reply app.json]: app.json has been updated

File: heroku (hunk 2 above)
Comment (tchan408) [reply heroku]: Yes, necessary update has been done. Thank you

**Review by lornajane — APPROVED** [review]
This is in good shape! Merging :+1:

**Pull request merged**